processing/src/main/java/io/druid/segment/StringDimensionIndexer.java

```diff
@@ -539,7 +546,70 @@ public int lookupId(String name)
   }
 
   @Override
-  public Object convertUnsortedEncodedArrayToActualArrayOrList(int[] key, boolean asList)
+  public LongColumnSelector makeLongColumnSelector(
+      IncrementalIndexStorageAdapter.EntryHolder currEntry, IncrementalIndex.DimensionDesc desc
+  )
+  {
+     return ZeroLongColumnSelector.instance();
```

Comment:
I believe QueryableIndexStorageAdapter does the same thing, so at least it's consistent with that. However, it'd be nice (for schema evolution of a column from string -> numeric) for the behavior to change to try to convert the string to a number.